src/scripts/main.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
-// write code here
+const people = document.querySelectorAll('li');
+const list = document.querySelector('ul');
+
+function change(salary) {
+  return +salary.slice(1).replaceAll(',', '');
+};
+
+const sortList = listToSort => {
+  const currentList = Array.prototype.slice.call(listToSort, 0);
```

Comment:
what is the purpose of it? you wanted to make a copy or what?)  Because listToSort is definitely equals currentList  now. Consider to remove it or use JSON.parse/stringify to make a copy
```suggestion
  const currentList = Array.prototype.slice.call(listToSort, 0);
```

Reply:
I have tried to use method 'sort', but it dosn`t work with NoteList, I found this solution in the Internet? but I guess it works the same as [...ListToSort].

Reply:
Done